OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -226,6 +226,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
     // Coordinate Set
     const CoordinateSet& getCoordinateSet() const {return get_CoordinateSet();}
 
+    /** Get the Coordinate(s) associated with this Joint. If there is only one
+        Coordinate, you can call getCoordinate() with no argument. */
+    const Coordinate& getCoordinate() const;
+    const Coordinate& getCoordinate(unsigned idx) const;
```

Comment:
This second method will not have a doxygen comment. So, the comment above will seem weird to users.